packages/client/components/AddTeamMemberModal.tsx

```diff
@@ -98,34 +117,59 @@ const AddTeamMemberModal = (props: Props) => {
   const {closePortal, meetingId, teamMembers, teamId} = props
   const [pendingSuccessfulInvitations, setPendingSuccessfulInvitations] = useState([] as string[])
   const [successfulInvitations, setSuccessfulInvitations] = useState<string[] | null>(null)
+  const [isSubmitted, setIsSubmitted] = useState(false)
   const [rawInvitees, setRawInvitees] = useState('')
   const [invitees, setInvitees] = useState([] as string[])
   const {error, onCompleted, onError, submitMutation, submitting} = useMutationProps()
   const atmosphere = useAtmosphere()
   const onChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+    if (isSubmitted) setIsSubmitted(false)
     const nextValue = e.target.value
     if (rawInvitees === nextValue) return
-    const parsedInvitees = parseEmailAddressList(nextValue)
+    const {parsedInvitees, invalidEmailExists} = parseEmailAddressList(nextValue) as any
```

Comment:
it'd be good to do the typescript casting within the parseEmailAddressList instead of outside of it.
that way you can reuse the function anywhere & you still get type safety